src/Hooks/UseIsInitial.ts

```diff
@@ -17,7 +17,7 @@ export const useIsInitial = (): boolean => {
   const navigation = useNavigation()
   const isInitial = useMemo(() => {
     const state = navigation.getState()
-    return calculateIsInitial(state, route)
+    return state != null ? calculateIsInitial(state, route) : false
```

Comment:
are we sure that if the state is missing we would like to tell it's not initial, let's double check